fiftyone_devicedetection_cloud/tests/51Degrees.csv

```diff
@@ -0,0 +1 @@
ded","ScreenInchesHeight","ScreenInchesSquare","ScreenInchesWidth","ScreenMMDiagonal","ScreenMMDiagonalRounded","ScreenMMHeight","ScreenMMSquare","ScreenMMWidth","ScreenPixelsHeight","ScreenPixelsWidth","ScreenType","SecondBackCameraMegaPixels","SecondFrontCameraMegaPixels","SecondScreenInchesDiagonal","SecondScreenInchesDiagonalRounded","SecondScreenInchesHeight","SecondScreenInchesSquare","SecondScreenInchesWidth","SecondScreenMMDiagonal","SecondScreenMMDiagonalRounded","SecondScreenMMHeight","SecondScreenMMSquare","SecondScreenMMWidth","SecondScreenPixelsHeight","SecondScreenPixelsWidth","SoC","SoCDesigner","SoCModel","SpecificAbsorbtionRateEU","SpecificAbsorbtionRateUS","SupportedBearers","SupportedBluetoothVersion","SupportedCameraFeatures","SupportedChargerTypes","SupportedI/O","SupportedSIMCardTypes","SupportedSensorTypes","Supports24p","SupportsPhoneCalls","SupportsWiDi","TAC","WeightWithBattery","WeightWithoutBattery","CcppAccept","PlatformDiscontinuedAge","PlatformDiscontinuedMonth","PlatformDiscontinuedYear","PlatformName","PlatformPreviewAge","PlatformPreviewMonth","PlatformPreviewYear","PlatformRank","PlatformReleaseAge","PlatformReleaseMonth","PlatformReleaseYear","PlatformVendor","PlatformVersion","SoftwareAudioCodecsDecode","SoftwareAudioCodecsEncode","SoftwareVideoCodecsDecode","SoftwareVideoCodecsEncode","StreamingAccept","AjaxRequestType","AnimationTiming","BlobBuilder","BrowserAudioCodecsDecode","BrowserAudioCodecsEncode","BrowserDiscontinuedAge","BrowserDiscontinuedMonth","BrowserDiscontinuedYear","BrowserFamily","BrowserName","BrowserPreviewAge","BrowserPreviewMonth","BrowserPreviewYear","BrowserPropertySource","BrowserRank","BrowserReleaseAge","BrowserReleaseMonth","BrowserReleaseYear","BrowserVendor","BrowserVersion","BrowserVideoCodecsDecode","BrowserVideoCodecsEncode","Canvas","CookiesCapable","CssBackground","CssBorderImage","CssCanvas","CssColor","CssColumn","CssFlexbox","CssFont","CssGrid","CssImages","CssMediaQueries","CssMinMax","CssOverflow","CssPosition","CssText","CssTransforms","CssTransitions","CssUI","DataSet","DataUrl","DeviceOrientation","Fetch","FileReader","FileSaver","FileWriter","FormData","Fullscreen","GeoLocation","History","Html-Media-Capture","Html5","Html5Audio","Html5Video","HtmlVersion","Http2","HttpLiveStreaming","Iframe","InVRMode","IndexedDB","IsDataMinimising","IsEmailBrowser","IsEmulatingDesktop","IsEmulatingDevice","IsWebApp","Javascript","JavascriptCanManipulateCSS","JavascriptCanManipulateDOM","JavascriptGetElementById","JavascriptPreferredGeoLocApi","JavascriptSupportsEventListener","JavascriptSupportsEvents","JavascriptSupportsInnerHtml","JavascriptVersion","Jpeg2000","Json","LayoutEngine","Masking","Meter","PostMessage","Preload","Progress","Promise","Prompts","Selector","SetHeaderBrowserAccept-CH","SetHeaderHardwareAccept-CH","SetHeaderPlatformAccept-CH","SupportsTls/Ssl","SupportsWebGL","Svg","TouchEvents","Track","Video","Viewport","WebP","WebWorkers","Xhr2","CrawlerName","IsCrawler"
```

Comment:
this file is actually downloaded during [ci/fetch-assets.ps1](https://github.com/51Degrees/device-detection-python/blob/main/ci/fetch-assets.ps1#L58C142-L59C2) - no need to ship it as part of the repo